packages/unmock-core/src/options.ts

```diff
@@ -11,6 +12,11 @@ export enum Mode {
 
 export class UnmockOptions implements IUnmockOptions {
   public save: boolean | string[] = true;
+  public actions: ActionsV0 = [
```

Comment:
Typing wise, this is not as clean IMO as e.g. `ActionsV0 | ActionsV0[]` would be.